utils/install-open-shift-tornjak.sh

```diff
@@ -0,0 +1,234 @@
+apiVersion: v1
+metadata:
+  name: $SPIRESCC
+allowHostDirVolumePlugin: true
+allowPrivilegedContainer: true
+allowHostPorts: true
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+fsGroup:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+groups:
+- system:authenticated
+EOF
+oc describe scc $SPIRESCC
+
+# get ingress information:
+ING=$(ibmcloud oc cluster get --cluster "$CLUSTERNAME" --output json | jq -r '.ingressHostname')
+INGSEC=$(ibmcloud oc cluster get --cluster "$CLUSTERNAME" --output json | jq -r '.ingressSecretName')
+INGSTATUS=$(ibmcloud oc cluster get --cluster "$CLUSTERNAME" --output json | jq -r '.ingressStatus')
+ibmcloud oc cluster get --cluster "$CLUSTERNAME" --output json | jq -r '.ingressMessage'
+
+
+# add the certs and keys
+#here=$(pwd)
+#cd /Users/sabath/workspace/mrsabath/tornjak/sample-keys
+# ./gen_domain.sh *.tsi-roks02-5240a919746a818fd9d58aa25c34ecfe-0000.eu-de.containers.appdomain.cloud $CLUSTERNAME
+#$SCRIPT_PATH/gen_domain.sh "*.$ING" $CLUSTERNAME
+echo "Generating certs..."
+openssl req -new -x509 -sha256 -key "$SCRIPT_PATH/../sample-keys/key.pem" -subj "/C=US/ST=CA/O=Acme, Inc./CN=example.com" -extensions SAN -config <(cat "$SCRIPT_PATH/../sample-keys/openssl.cnf" <(printf "[SAN]\nsubjectAltName=DNS:*.${ING},DNS:example.com,DNS:www.example.com")) -out "$SCRIPT_PATH/../sample-keys/$CLUSTERNAME.pem"
+# store the certs in the secret
+oc -n tornjak create secret generic tornjak-certs \
+ --from-file="$SCRIPT_PATH/../sample-keys/key.pem" \
+ --from-file=cert.pem="$SCRIPT_PATH/../sample-keys/$CLUSTERNAME.pem" \
+ --from-file=tls.pem="$SCRIPT_PATH/../sample-keys/$CLUSTERNAME.pem" \
+ --from-file=mtls.pem="$SCRIPT_PATH/../sample-keys/$CLUSTERNAME.pem"
+# cd $here
+
+# run helm install for the tornjak server
+helm install --set namespace=$PROJECT --set clustername=$CLUSTERNAME --set trustdomain=$TRUSTD tornjak charts/tornjak # --debug
+helm list
+
+# oc -n $PROJECT expose svc/$SPIRESERVER
+# Ingress route for spire-server
+oc -n $PROJECT create route passthrough --service spire-server
+oc -n $PROJECT get route
+INGRESS=$(oc -n $PROJECT get route spire-server -o jsonpath='{.spec.host}{"\n"}')
+echo $INGRESS
+
+# setup TLS secret:
+CRN=$(ibmcloud oc ingress secret get -c "$CLUSTERNAME" --name "$INGSEC" --namespace openshift-ingress --output json | jq -r '.crn')
+ibmcloud oc ingress secret create --cluster "$CLUSTERNAME" --cert-crn "$CRN" --name "$INGSEC" --namespace $PROJECT
+if [ "$?" == "0" ]; then
+  echo "All good"
+fi
+
+# create ingress deployment:
+oc create -f- <<EOF
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: spireingress
+spec:
+  tls:
+  - hosts:
+    - $INGRESS
+    secretName: $INGSEC
+  rules:
+  - host: $INGRESS
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: spire-server
+          servicePort: 8081
+EOF
+
+# create route for Tornjak TLS:
+oc -n $PROJECT create route passthrough tornjak-tls --service tornjak-tls
```

Comment:
maybe have a macro here between each call to just check and exit if non-success? so that we can catch errors?  This will help catch any changes in `oc` CLI api